prboom2/src/m_menu.c

```diff
@@ -168,15 +168,17 @@ extern dboolean  message_dontfuckwithme;
 // displayed
 
 dboolean setup_active      = false; // in one of the setup screens
+dboolean set_general_active = false;
 dboolean set_keybnd_active = false; // in key binding setup screens
+dboolean set_display_active = false;
+dboolean set_demos_active = false; // in demos setup screen
+dboolean set_compatibility_active = false;
 dboolean set_weapon_active = false; // in weapons setup screen
-dboolean set_status_active = false; // in status bar/hud setup screen
 dboolean set_auto_active   = false; // in automap setup screen
+dboolean level_table_active = false;
```

Comment:
We can make these variables `static` so that they are only visible in the m_menu.c file. We can consider making most of the variables and function declarations `static`, since there are a lot of changes in m_menu.c anyway.

By the way, how do you want to merge this PR? Squash or history editing? There are a lot of commits.

Reply:
I agree. Did a quick pass to make most of the variables and functions static in https://github.com/kraflab/dsda-doom/pull/592/commits/e835bb85f3831bf74eed9f735301da0886e90936

I wanted to simply squash, but since there's this big change, went with history editing